src/main/java/core/basesyntax/Application.java

```diff
@@ -3,5 +3,14 @@
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+
+        Ball firstRandomBall = lottery.getRandomBall();
+        Ball secondRandomBall = lottery.getRandomBall();
+        Ball thirdRandomBall = lottery.getRandomBall();
```

Comment:
Use a for loop for creating several objects of the same class instead of creating them individually. This makes the code more scalable and maintainable.